dashboard/test/testing/includes_metrics.rb

```diff
@@ -49,7 +49,7 @@ def initialize(metric_name, dimensions)
       def matches?(available_parameters)
         actual_metrics = available_parameters.shift
         @expected_dimensions.all? do |expected_dimension_name, expected_value|
-          sought_metric = actual_metrics.find {|m| m[:metric_name] == @expected_metric_name}
+          sought_metric = actual_metrics.find {|m| m[:metric_name].to_sym == @expected_metric_name}
```

Comment:
This and the other addition of `to_sym` were necessary to support Metrics with String names.